tests/periph_gpio_arduino/main.sketch

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2017 Tom Keddie
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Manual test application for GPIO peripheral drivers using Arduino pin numbering
```

Comment:
This line is really long, please trim to 80 characters

Reply:
Thanks for the review, the code is base on the other gpio test, I dropped the copyright as requested, it's not clear to me which path to take here so advice is appreciated.